src/material-table.js

```diff
@@ -456,7 +456,7 @@ export default class MaterialTable extends React.Component {
 
     return (
       <DragDropContext onDragEnd={this.onDragEnd}>
-        <props.components.Container style={{ position: 'relative', ...props.style }}>
+        <props.components.Container className={props.className} style={{ position: 'relative', ...props.style }}>
```

Comment:
I thought of doing this as well, adding the  className prop to make the whole thing more similar to Mui. But in this specific case you _could_ just supply a custom Container component instead. :)

Reply:
I didn't think about overriding the Container component.  Maybe this should be stated in the styling section in the documentation.

Reply:
Just another thought.  If someone were using several material tables in their project, it would make more sense to have a class for styling.

Reply:
Ah, I agrew with you. It would make a lot of sense to swap most of the custom component props for mui overrides instead! Also, I usually just put a thin wrapper component around MaterialTable, and expose that to the rest of the app with all my preferred settings :)